src/Place/ReadModel/History/HistoryProjector.php

```diff
@@ -0,0 +1,135 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\History;
+
+use Broadway\Domain\DomainMessage;
+use CultuurNet\UDB3\Cdb\EventItemFactory;
+use CultuurNet\UDB3\History\BaseHistoryProjector;
+use CultuurNet\UDB3\History\Log;
+use CultuurNet\UDB3\Place\Events\DescriptionTranslated;
+use CultuurNet\UDB3\Place\Events\LabelAdded;
+use CultuurNet\UDB3\Place\Events\LabelRemoved;
+use CultuurNet\UDB3\Place\Events\PlaceCreated;
+use CultuurNet\UDB3\Place\Events\PlaceDeleted;
+use CultuurNet\UDB3\Place\Events\PlaceImportedFromUDB2;
+use CultuurNet\UDB3\Place\Events\PlaceUpdatedFromUDB2;
+use CultuurNet\UDB3\Place\Events\TitleTranslated;
+use DateTime;
+use DateTimeZone;
+
+final class HistoryProjector extends BaseHistoryProjector
+{
+
+    public function handle(DomainMessage $domainMessage)
+    {
+        $event = $domainMessage->getPayload();
+        switch (true) {
+            case $event instanceof PlaceCreated:
+                $this->projectPlaceCreated($event, $domainMessage);
+                break;
+            case $event instanceof PlaceDeleted:
+                $this->projectPlaceDeleted($event, $domainMessage);
+                break;
+            case $event instanceof LabelAdded:
+                $this->projectLabelAdded($event, $domainMessage);
+                break;
+            case $event instanceof LabelRemoved:
+                $this->projectLabelRemoved($event, $domainMessage);
+                break;
+            case $event instanceof DescriptionTranslated:
+                $this->projectDescriptionTranslated($event, $domainMessage);
+                break;
+            case $event instanceof TitleTranslated:
+                $this->projectTitleTranslated($event, $domainMessage);
+                break;
+            case $event instanceof PlaceImportedFromUDB2:
+                $this->projectPlaceImportedFromUDB2($event, $domainMessage);
+                break;
+            case $event instanceof PlaceUpdatedFromUDB2:
+                $this->projectPlaceUpdatedFromUDB2($event, $domainMessage);
+                break;
+        }
+    }
+
+    private function projectPlaceCreated(PlaceCreated $event, DomainMessage $domainMessage): void
+    {
+        $this->writeHistory(
+            $event->getPlaceId(),
+            $this->createGenericLog($domainMessage, 'Aangemaakt in UiTdatabank')
+        );
+    }
+
+    private function projectPlaceDeleted(PlaceDeleted $event, DomainMessage $domainMessage): void
+    {
+        $this->writeHistory(
+            $event->getItemId(),
+            $this->createGenericLog($domainMessage, 'Aangemaakt verwijderd')
```

Comment:
This message should be `Place verwijderd`. `Aangemaakt verwijderd` means `Created deleted`. 😉 
I'm not sure we had to add this event handler already, as the history projector for events doesn't handle `EventDeleted` either, but we can leave it in now.